src/librustc_metadata/cstore_impl.rs

```diff
@@ -60,7 +60,7 @@ macro_rules! provide {
 
                 $tcx.dep_graph.read(dep_node);
 
-                let $cdata = $tcx.sess.cstore.crate_data_as_rc_any($def_id.krate);
+                let $cdata = $tcx.crate_data_as_rc_any($def_id.krate);
```

Comment:
I was worried about this but I think it's fine - this will eventually be just a byte buffer that the query system interprets.